src/sentry/receivers/email.py

```diff
@@ -7,7 +7,7 @@
 def create_email(instance, created, **kwargs):
     if created:
         try:
-            with transaction.atomic():
+            with transaction.atomic(using=router.db_for_write(Email)):
```

Comment:
This was really awful to trace down, but was causing errors in split DB mode with `You can't execute queries until the end of the 'atomic' block` exceptions due to the transaction wrapping around this.